src/main/resources/org/rust/ide/colors/highlighterDemoText.rs

```diff
@@ -23,6 +23,11 @@ struct <STRUCT>Object</STRUCT><<TYPE_PARAMETER>T</TYPE_PARAMETER>> {
     <FIELD>fields</FIELD>: <STRUCT>HashMap</STRUCT><<TYPE_PARAMETER>T</TYPE_PARAMETER>, <PRIMITIVE_TYPE>u64</PRIMITIVE_TYPE>>
 }
 
+union <UNION>MyUnion</UNION> {
```

Comment:
```suggestion
<KEYWORD>union</KEYWORD> <UNION>MyUnion</UNION> {
```
`union` is a contextual keyword in Rust. It cannot be highlighted via lexer so it isn't hightlighted automatically in setting highlighting demo. Let's add an explicit color tag for `union` here to get proper highlighting for it